package/upgrade/do_upgrade_node.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+until $SCRIPT_DIR/upgrade_node.sh $@; do
+  # Do not fail pre-drain/post-drain jobs
+  case $1 in
+    "pre-drain"|"post-drain")
+	;;
+    *)
+	exit 1
+	;;
+  esac
+
+  # A fallback to exit if seeing some files
+  if [ -e "/tmp/skip-retry-with-fail" ]; then
+    exit 1
+  fi
+
+  if [ -e "/tmp/skip-retry-with-succeed" ]; then
+    exit 0
+  fi
```

Comment:
I'm looking for these two temp files but cannot find any. When are these two temp files created? Thanks.

Reply:
They are just there in case we want to continue the jobs:)
So we can shell into the pod and touch the file.

Reply:
Cool, thanks for the explanation!